### ISSUE TITLE
Add Package does not seem to use the source defined in paket.dependencies

### ISSUE THREAD

**Issue by theduffmtl** [fsprojects/Paket.VisualStudio#89]
how can i configure the add packet : package Manager popup to use the sources defined in my paket.dependencies . I need to install internal packages that are not on nuget.org but on private repo ?

**Comment by forki** [comment]
do you mean the search function in the dialog?

**Comment by forki** [comment]
this https://github.com/fsprojects/Paket/issues/1298?

**Comment by theduffmtl** [comment]
yes it the same but as a quicker solution it would be nice at leat to be able to add package -> set the name of the package -> ok -> the paket install command is triggered. to sum up have visual studio interface to launch paket install without the search.

**Comment by dcorriveau-omniscient** [comment]
+1 for this.
Having a UI to add packages directly from within VS is a requirement in our environment and since we're using a private Nuget server this issue is a show stopper.
Thanks for the quick replies / fixes.

**Comment by forki** [comment]
The corresponding paket issue is marked as up-for-grabs and deliberately left open for people to jump in. It's not that hard to implement and @cdrnet is already looking into it

**Comment by forki** [comment]
I added support for custom feeds, please test it and let me know

**Comment by dcorriveau-omniscient** [comment]
Amazing !
The UI still need some love but it's usable. Thanks.
